### PR TITLE
[Notifier] Added Iterable notifier

### PR DESCRIPTION
**Pull request by balazscsaba2006**
Added Iterable notifier bridge to the docs.

### CODE REVIEW

**Comment by OskarStark** [comment]
Please target 5.4 branch

File: notifier.rst [hunk 1]
```diff
@@ -182,7 +183,7 @@ Zulip           ``symfony/zulip-notifier``            ``zulip://EMAIL:TOKEN@HOST
 
 .. versionadded:: 5.3
 
-    The Gitter, Mercure, FakeChat and Microsoft Teams integrations were introduced in Symfony 5.3.
+    The Gitter, Mercure, FakeChat, Microsoft Teams and Iterable integrations were introduced in Symfony 5.3.
```
Comment (OskarStark) [inline notifier.rst]: Please use versionadded 5.4

**Comment by balazscsaba2006** [comment]
@OskarStark I've made the changes, thank you.

**Comment by javiereguiluz** [comment]
Closing because sadly the related code PR was closed: https://github.com/symfony/symfony/pull/41713

**Pull request closed without merging**